Source/Scene/Scene.js

```diff
@@ -471,18 +470,6 @@ function Scene(options) {
    */
   this.debugShowFramesPerSecond = false;
 
-  /**
-   * This property is for debugging only; it is not for production use.
-   * <p>
-   * Displays depth information for the indicated frustum.
-   * </p>
-   *
-   * @type Boolean
-   *
-   * @default false
-   */
-  this.debugShowGlobeDepth = false;
```

Comment:
Since this property is part of the public API removing it counts as a breaking change and should be mentioned in `CHANGES.md`. I don't think we need a deprecation period since it is for debugging only and has been broken for a while now.